### ISSUE TITLE
Typos, minor corrections, and general feedback

### ISSUE THREAD

**Issue by agitter** [carpentries-incubator/deep-learning-intro#207]
I read through the materials to see whether they might be useful for my teaching or student rotation research projects.  This looks very well done overall.  I'm leaving a few minor comments as well as non-essential general thoughts.

Minor corrections and typos:
- The first image in the Introduction episode is broken
- Introduction episode is missing a solution to item 5 in the Deep Learning Problems Exercise
- Episode 2 "This is a commonly used activation **functions**" should be singular
- Episode 3 explore the dataset exercise "**variabel** types"
- Episode 3 "Split data into training, validation, and test set" exercise has an extra "**`**" to format code
- Episode 3 "to **splot** if"
- Episode 3 "stop training **of** the"
- Episode 4 solution "one **signal** data point"
- Episode 4 solution "add **a** an extra"
- "**networt**" instead of "network" twice near the end of the Lesson Design page

General thoughts:
- The Jax framework is gaining popularity and could have a sentence or two in the introduction section about different libraries.
- PyTorch Lightning can make many types of PyTorch workflows more convenient and reduce boilerplate code. That could be relevant in the library discussion.
- In Episode 3, the "Watch your model training closely" section describes using a validation set to avoid "iterating through countless cycles of model trainings and subsequent evaluations with a reserved test set". That gives the impression that the validation set is being used for convenience.  It seems important to stress (in the episode or instructor notes) that using the test set repeated to assess model or hyperparameter performance is an invalid workflow because the model will be indirectly fit to the test set.  Then the test set can no longer be used to estimate generalization performance on new data, which is usually the real goal.

**Comment by dafnevk** [comment]
Thanks a lot, these are all valid points. I created separate issues for the different points and will close this issue.
